deploy/csi/kms-config.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: v1
+kind: ConfigMap
+data:
+  config.json: |-
+    {
+      "vault-test": {
+        "encryptionKMSType": "vault",
+        "vaultAddress": "http://vault.default.svc.cluster.local:8200",
+        "vaultAuthPath": "/v1/auth/kubernetes/login",
+        "vaultRole": "csi-kubernetes",
+        "vaultPassphraseRoot": "/v1/secret",
+        "vaultPassphrasePath": "ceph-csi/",
+        "vaultCAVerify": "false"
+      }
+    }
+metadata:
+  name: ceph-csi-encryption-kms-config
+  namespace: ceph-csi
```

Comment:
why do we need this file?

Reply:
I'm not fully aware of the details but as far as I know, ksm (key management system) is needed to generate a passphrase for each encrypted rbd image. 

https://github.com/nixpanic/ceph-csi/blob/61c56576dd441aedbd954148982b585c467972cf/docs/deploy-rbd.md#encryption-kms-configuration

Reply:
Currently we don't set or use encryption for rbd image, so we don't need it.